web/themes/custom/pineapple/templates/webform/webform--contact.html.twig

```diff
@@ -0,0 +1,32 @@
+{#
+/**
+ * @file
+ * Theme implementation for a 'webform' element.
+ *
+ * This is an copy of the webform.html.twig theme_wrapper which includes the
+ * 'title_prefix' and 'title_suffix' variables needed for
+ * contextual links to appear.
+ *
+ * Available variables
+ * - attributes: A list of HTML attributes for the wrapper element.
+ * - children: The child elements of the webform.
+ * - title_prefix: Additional output populated by modules, intended to be
+ *   displayed in front of the main title tag that appears in the template.
+ * - title_suffix: Additional output populated by modules, intended to be
+ *   displayed after the main title tag that appears in the template.
+ *
+ * @see template_preprocess_webform()
+ * @see _webform_form_after_build()
+ *
+ * @ingroup themeable
+ */
+#}
+{% include "@components/_heading.twig" with {
+  heading_text: title,
+  heading_level: 2
+} %}
+<form class="contact-form">
```

Comment:
It's not necessary to add a class here like this, if you add attributes back in you'll see there are plenty of classes available for you to use

Reply:
done